sources/core/Stride.Core.Yaml.Tests/Serialization/SerializationTests2.cs

```diff
@@ -1506,21 +1510,6 @@ public class ObjectWithMask
             internal int Int3 { get; set; }
         }
 
-        [Fact]
-        public void TestImplicitMemberType()
-        {
-            var settings = new SerializerSettings() {LimitPrimitiveFlowSequence = 0};
-
-            var text = @"!ClassWithImplicitMemberType
-Test:
-  String: test
-";
-
-            settings.RegisterTagMapping("ClassWithImplicitMemberType", typeof(ClassWithImplicitMemberType));
-            settings.RegisterTagMapping("ClassWithImplicitMemberTypeInner", typeof(ClassWithImplicitMemberTypeInner));
-            SerialRoundTrip(settings, text);
-        }
-
```

Comment:
This test has been removed as omitting object type when they match the type at initialization but diverge from the member's return type has been deprecated in a previous pr.